docs/konnectors-dev.md

```diff
@@ -0,0 +1,72 @@
+[Table of contents](README.md#table-of-contents)
+
+
+# Develop a konnector
+
+
+
+## In standalone 
+
+This allows to run a konnector without a cozy-stack available. This is probably
+the best option for the early development. 
+
+You can find a detailed explanation with a video inside the [konnector getting started](https://docs.cozy.io/en/tutorials/konnector/getting-started/)
+page.
+
+
+
+## Run in local and save in a stack
+
+This method requires a working cozy-stack. It will allows you to run the konnector
+outside the cozy-stack but saves the data inside the cozy-stack. This can be usefull
+when your konnector is not available inside the registry but you still want to save
```

Comment:
```suggestion
when your konnector is not available in one of the registries but you still want to save
```

Reply:
@Peltoche it looks like you have missed some suggestions made by Erwan